fastapi_websocket_rpc/websocket_rpc_client.py

```diff
@@ -114,17 +120,17 @@ async def connect(self, uri: str, **connect_kwargs):
         except ConnectionRefusedError:
```

Comment:
`websockets` is the name of the default library used to handle websockets.
`websocket-client` is the name of the optional library that supports HTTP proxies. But its import name is `websocket`.
`self._websocket` is the underlying websocket that gets created during connection either in `WebSocketsClientHandler` or `ProxyEnabledWebSocketClientHandler` class.

As I mentioned in the previous comments, even if one of the mentioned libraries is missing, the file should be importable. So the proposed import scheme is to try to import the library and not fail otherwise. But if a handler is called and the necessary library is not installed, then fail in runtime. That's what I suggested in this PR.
If you find it unreasonable I can change to a better implementation.

**P. S.** I was inspired by the logic of `aiohttp` DNS resolver implementations - https://github.com/aio-libs/aiohttp/blob/v3.11.11/aiohttp/resolver.py#L90-L91